src/app/pages/table-page/basic-table.component.ts

```diff
@@ -1,38 +1,46 @@
 import {
   CUSTOM_ELEMENTS_SCHEMA,
   Component,
-  ElementRef,
-  ViewChild,
-  inject,
-  Renderer2,
-  AfterViewInit, Input
-} from '@angular/core'
-import {TableData} from './table-data'
+} from '@angular/core';
+import exampleData from './exampleData.json';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-basic-table',
   template: `
     <h1>Basic Table</h1>
-    <tds-table vertical-dividers="false" compact-design="false" enable-responsive="false">
+    <tds-table vertical-dividers="false" compact-design="false" responsive="false">
```

Comment:
I would personally leave out the attributes entirely instead of assigning them a falsy value and expect that to work